common/create-store.js

```diff
@@ -6,6 +6,20 @@ const {Channel} = require("common/ReduxChannel");
 const parseUrlMiddleware = require("common/parse-url-middleware");
 const loggerMiddleware = require("common/redux-logger");
 const {LOCAL_STORAGE_KEY} = require("common/constants");
+let store;
```

Comment:
This is kind of a departure from how `createStore` worked originally (i.e., you could create multiple  independent stores with no overlapping state),  although in practice right now we only have one store per context so it won't actually be a problem. I think it might be worth commenting at least though